### PR TITLE
Set value of Radio as undefined instead of "undefined"

### PR DESCRIPTION
**Pull request by anserwaseem**
## Describe your changes
Fix `value` of Radio being initialized as "undefined"
### Screenshots [Optional]
Before:
image
After:
image

## Issue ticket number and link

Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] I have added tests
- [x] I have added a changeset `pnpm changeset add`
- [x] I have added example usage in the kitchen sink app

### CODE REVIEW

File: packages/runtime/src/widgets/Form/Radio.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ export const RadioWidget: React.FC<RadioWidgetProps> = (props) => {
   const formInstance = Form.useFormInstance();
 
   useEffect(() => {
-    setValue(String(values?.initialValue));
+    setValue(values?.initialValue);
```
Comment (sagardspeed2) [inline packages/runtime/src/widgets/Form/Radio.tsx:48]: @anserwaseem if you remove this type casting here, the initial value, I mean the initial selected radio button will not be selected, you need to click two times to reflect the UI changes.

Please test this also.

**Review by sagardspeed2 — CHANGES_REQUESTED** [review]

File: packages/runtime/src/widgets/Form/Radio.tsx (hunk 1 above)
Comment (sagardspeed2) [reply packages/runtime/src/widgets/Form/Radio.tsx:48]: image
@anserwaseem check this, as radio button has assigned initial value but not rendered on UI

Comment (anserwaseem) [reply packages/runtime/src/widgets/Form/Radio.tsx]: i missed that, ty.

**Review by evshi — APPROVED** [review]

**Comment by evshi** [comment]
@anserwaseem @sagardspeed2 anything outstanding on this one or can we close it out?

**Review by sagardspeed2 — APPROVED** [review]

**Comment by sagardspeed2** [comment]
Looking good for me

**Pull request merged**